stdlib/Markdown/src/Markdown.jl

```diff
@@ -40,6 +40,11 @@ function docexpr(source::LineNumberNode, mod::Module, s, flavor = :julia)
     :($doc_str($(mdexpr(s, flavor)), $(QuoteNode(source)), $mod))
 end
 
+"""
+    @md_str -> MD
+
+Construct a `MD` Markdown object from the given string.
```

Comment:
```suggestion
Parse the given string as Markdown text and return a corresponding [`Markdown.MD`](@ref) object.
```

Reply:
Thanks! I've used your wording, but I've removed the `(@ref)` as this didn't compile. I'm assuming that this would have required to also add the `Markdown.MD` type into the docs which seemed excessive to me.